home-manager/gnome.nix

```diff
@@ -237,18 +211,27 @@ in
 
       "org/gnome/desktop/wm/preferences" = {
         theme = "Nordic";
+
+        num-workspaces = 3;
+        workspace-names = [
+          "Main"
+          "Sandbox"
+          "Music"
+        ];
```

Comment:
This cannot be persisted, because of paperwm saves it with uuid

```
/org/gnome/desktop/wm/preferences/workspace-names
  ['Main', 'Workspace 2', 'Musi']

/org/gnome/shell/extensions/paperwm/workspaces/5719a18c-cc5e-44a8-88a7-f6571fa0236e/name
  'Music'

/org/gnome/desktop/wm/preferences/workspace-names
  ['Main', 'Workspace 2', 'Music']
```

https://github.com/paperwm/PaperWM/issues/720#issuecomment-1826842508